### DISCUSSION TITLE
Tidal API is read-only?

### DISCUSSION THREAD

**Discussion by derezzolution** [tidal-music/discussions#39] [Q&A]
I've checked out the documentation in https://developer.tidal.com/apiref and, if I'm reading this correctly, the Tidal API is immutable only, right? I had hoped to find an API where I could create a new Tidal playlist and add Tidal tracks to it, but it sounds like this isn't something that would currently be possible with the API as is stands. Couple questions:

1. Is my understanding correct?
2. Where in the roadmap will mutable APIs be available?

Thanks so much!

PS: Sorry if this is the wrong location to post. I wasn't able to find concise details anywhere and Tidal support won't talk to me. 🥴

**Comment by stefanmoro** [comment]
Hey @derezzolution ,

Please see #6 for more information about playlist API.

**Reply by derezzolution** [reply]
Thanks for linking in that feature request @stefanmoro! 🙏
